controller/signin.js

```diff
@@ -0,0 +1,25 @@
+import { v4 as uuidv4 } from 'uuid';
+
+uuidv4();
+import users from '../config/user.json' assert{type: 'json'};
```

Comment:
`import users from '../config/user.json'` should be above the `uuidv4` initialization

Please remove `assert{type: 'json'}`